firewood/src/merkle.rs

```diff
@@ -1471,7 +1412,7 @@ mod tests {
         let mut merkle = Merkle::new(HashedNodeStore::initialize(MemStore::new(vec![])).unwrap());
         for (k, v) in items.iter() {
             merkle.insert(k.as_ref(), Box::from(v.as_ref()))?;
-            println!("{}", merkle.dump()?);
+            // println!("{}", merkle.dump()?);
```

Comment:
Interesting question whether we want this or not

We can consider moving it to debug logging, but it was super handy during insert testing!

Reply:
uncommented. Let's leave it for now